app/models/patient.rb

```diff
@@ -129,6 +129,136 @@ def latest_transfer
       .where('latest_assessment_at < ? OR latest_assessment_at IS NULL', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)
   }
 
+  # Patients who are eligible for reminders
+  #
+  # GENERAL SCOPE OVERVIEW:
+  # - Everything is before the OR is essentially checking if the HoH or the patient
+  #   who isn't in a household is eligible on their own.
+  # - Everything within the OR is checking if the HoH has any DEPENDENTS (excluding self)
+  #   that would make the HoH eligible to receive notifications for them.
+  scope :better_reminder_eligible, lambda {
+    joins(:dependents)
+      .monitoring_open
+      .within_preferred_contact_time
+      .has_not_reported_recently_enough_for_reminder_exlusion
+      .should_be_monitoring
+      .has_usable_preferred_contact_method
+      .where('patients.id = patients.responder_id')
+      .where(pause_notifications: false)
+      .where('patients.last_assessment_reminder_sent <= ? OR patients.last_assessment_reminder_sent IS NULL', 12.hours.ago)
+      .or(hoh_eligible_for_any_dependents)
+  }
+
+  # Non-eligible contact methods are: ['Unknown', 'Opt-out', '', nil]
+  scope :has_usable_preferred_contact_method, lambda {
+    where.not(preferred_contact_method: ['Unknown', 'Opt-out', '', nil])
+  }
+
+  # Check if the HoH has any dependents that make them eligible to
+  # recieve notifications on the dependent's behalf.
+  # The joined table is referred to as `dependents_patients` and is used for all
+  # checks made on the dependents. Anywhere the `patients` table is specified or
+  # where a hash is used for a condition are checks made on the HoH.
+  #
+  # A dependent patient makes the HoH reminder eligible if:
+  # - HoH is not purged
+  # - HoH has a usable preferred contact method
+  # - HoH has not paused notifications
+  # - current HoH local time is within the preferred contact time window
+  # - patient has monitoring ON
+  # - patient is not purged
+  # - patient is in isolation
+  #   OR
+  #   patient is in continuous exposure
+  #   OR
+  #   patient last date of exposure is on or after (today - monitoring_period_days)
+  #   OR
+  #   patient last date of exposure is null and created at of exposure is on or after (today - monitoring_period_days)
+  scope :hoh_eligible_for_any_dependents, lambda {
```

Comment:
How about `has_eligible_dependents` instead?

Reply:
ok, changing this